components/brave_wallet_ui/components/asset-name-and-icon/index.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import { StyledWrapper, NameAndSymbolWrapper, AssetName, AssetSymbol, AssetIcon } from './style'
+
+export interface Props {
+  symbol: string
+  assetName: string
+  assetLogo: string
+}
+
+export const AssetNameAndIcon = (props: Props) => {
+  const { assetLogo, assetName, symbol } = props
+
+  return (
+    <StyledWrapper>
+      <AssetIcon src={`chrome://image?${assetLogo}`} loading="lazy" />
```

Comment:
do we know how long gets cached? could be a privacy problem. cc @thypon @kdenhartog

Reply:
also are the assetLogo's being proxied through https://github.com/brave-intl/bat-go/pull/1234 ?